client/state/ui/selectors/site-data.ts

```diff
@@ -11,13 +11,16 @@ export interface SiteData {
 	wpcom_url?: string;
 	jetpack?: boolean;
 	plan: SiteDataPlan;
+	products?: SiteDataPlan[];
 	capabilities?: Record< string, boolean >;
+	is_wpcom_atomic?: boolean;
```

Comment:
Added the missing fields here, otherwise we were getting incompatibility errors like this:

```
Conversion of type '(SiteData | null | undefined)[]' to type 'JetpackSite[]' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Type 'SiteData | null | undefined' is not comparable to type 'JetpackSite'.
    Property 'is_wpcom_atomic' is missing in type 'SiteData' but required in type 'JetpackSite'.ts(2352)
 ```